tests/sweep_framework/sweeps/reduction/traces/max_traces.py

```diff
@@ -15,24 +15,46 @@
 
 parameters = {
     "default": {
-        "height": [25],
-        "width": [4],
-        "dtype": [ttnn.float32, ttnn.bfloat16],
+        "height": [25, 16],
```

Comment:
Please revert the changes for the traces files (argmax_traces.py and max_traces.py). These are from traces, and should only change when traces change or new traces are added.